### ISSUE TITLE
Concatenate range to list

### ISSUE THREAD

**Issue by zuxfoucault** [WhitakerLab/BrainNetworksInPython#64]
Hi

I have a question regrading the code in this line [https://github.com/WhitakerLab/BrainNetworksInPython/blob/master/scripts/make_graphs.py#L90](https://github.com/WhitakerLab/BrainNetworksInPython/blob/master/scripts/make_graphs.py#L90), which I got:
![screen shot 2018-04-27 at 14 20 24](https://user-images.githubusercontent.com/3050531/39389317-bd776072-4ab9-11e8-99ad-5b16713d3a75.png)

**Comment by Islast** [comment]
This is fixed in #74, which is waiting merging to the dev branch, and will then wait some more before it is merged into the master branch. You may or may not find it easier to start working from the dev branch. Sorry for the long wait on this :cry:
